docs/schemas/Marmoset-Metadata/HMBA Marmoset Population.csv

```diff
@@ -0,0 +1,45 @@
+﻿Proposed BICAN Field Name,BICAN UUID,Label,Variable (Short) Name,Definition,Short Description,Human Readable Label,is_required,Permissible Values (incl Null),Description (Permissible Values),Data Type,Version Number,Version Date,Sub Domain Name,Domain Name,Size,Input Restrictions,Min Value,Max Value,Measurement Type
+Local Donor ID,,Local Donor ID,subject_id,,,,,,,text,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Sex at Birth,,Sex at Birth,sex,,,,,,,exclusive_categorical,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Age Value (Years),,Age Value (Years),age_of_death,,,,,,,numeric,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Date of Death,,Date of Death,year_of_death,,,,,,,date,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Manner of Death,,Manner of Death,manner_of_death,,,,,,,exclusive_categorical,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Mating Status,,Mating Status,mating_status,,,,,,,exclusive_categorical,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Medical Records Available,,Medical Records Available,medical_records_available,,,,,,,exclusive_categorical,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Hemisphere,,Hemisphere,hemisphere,,,,,,,exclusive_categorical,,,General Specimen Data,HMBA_Marmoset_Population,,,,,
+Post Mortem Interval,,Post Mortem Interval,post_mortem_interval,,,,,,,numeric,,,General Specimen Data,HMBA_Marmoset_Population,,,,,
+Left Hemisphere Preparation,,Left Hemisphere Preparation,left_hemisphere_preparation,,,,,,,exclusive_categorical,,,General Specimen Data,HMBA_Marmoset_Population,,,,,
+Left Hemisphere Preparation Specify,,Left Hemisphere Preparation Specify,left_hemisphere_preparation_specify,,,,,,,text,,,General Specimen Data,HMBA_Marmoset_Population,,,,,
+Right Hemisphere Preparation,,Right Hemisphere Preparation,right_hemisphere_preparation,,,,,,,exclusive_categorical,,,General Specimen Data,HMBA_Marmoset_Population,,,,,
+Right Hemisphere Preparation Specify,,Right Hemisphere Preparation Specify,right_hemisphere_preparation_specify,,,,,,,text,,,General Specimen Data,HMBA_Marmoset_Population,,,,,
+RIN,,RIN,rin,,,,,,,numeric,,,General Specimen Data,HMBA_Marmoset_Population,,,,,
+RINe,,RINe,rine,,,,,,,numeric,,,General Specimen Data,HMBA_Marmoset_Population,,,,,
+pH,,pH,ph,,,,,,,numeric,,,General Specimen Data,HMBA_Marmoset_Population,,,,,
+Brain Weight Measurement,,Brain Weight Measurement,brain_weight,,,,,,,numeric,,,General Specimen Data,HMBA_Marmoset_Population,,,,,

```

Comment:
Is it "Brain Weight Measurement" or just "Brain Weight"?

Reply:
if this is "Brain Weight" should we have somewhere also the units, as for other measurements?